hifi2genome/samplesheet_full_test.csv

```diff
@@ -0,0 +1,25 @@
+﻿sample,fastq
+acineobacter_baumanii_AYE,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2001.fq.gz 
+bacillus_cereus_971,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2002.fq.gz
+bacillus_subtilis_w23,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2003.fq.gz
+burkholderia_cepacia_ucb717,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2004.fq.gz
+burkholderia_multivorans_249,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2005.fq.gz
+enterococcus_faecalis_og1rf,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2006.fq.gz
+escherichia_coli_h10407,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2007.fq.gz
+escherichia_coli_k12_mg1655,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2008.fq.gz
+helicobacter_pylori_j99,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2009.fq.gz
+klebsiella_pneumonia_baa2146,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2010.fq.gz
+listeria_monocytogenes_li2,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2011.fq.gz
+listeria_monocytogenes_li23,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2012.fq.gz
+methanocorpusculum_labreanum_z,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2013.fq.gz
+neisseria_meningitidis_fam18,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2014.fq.gz
+neisseria_meningitidis_serogroup_b,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2015.fq.gz
+rhodopseudomonas_palustris_cga009,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2016.fq.gz
+salmonella_enterica_lt2,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2017.fq.gz
+salmonella_enterica_ty2,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2018.fq.gz
+staphylococcus_aureus_seattle1945,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2019.fq.gz
+staphylococcus_aureus_USA300_TCH1516,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2020.fq.gz
+streptococcus_pyogenes_bruno,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2021.fq.gz
+thermanaerovibrio_acidaminovorans_dsm6589,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2022.fq.gz
+treponema_denticola_a,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2023.fq.gz
+vibrio_parahaemolyticus_eb101,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2024.fq.gz
```

Comment:
nit: no new line at EOF

Reply:
I made a public bucket on our AWS account and put them there for now because I can't find this dataset publicly available, only on pacbio's website. So I downloaded the BAM files from their site, converted to fastq, then uploaded to a public bucket within our account. I'm assuming that's fine but maybe not for data reuse/license issues with pacbio and then having a public bucket through our AWS? For the latter it amounts to only ~2GB of data. There doesn't seem to be a license associated with the dataset on the Pacbio site.

Reply:
oh hm. i think it's fine. Maybe document in the readme where the data came from?